pkgs/development/python-modules/gmsh/fix_paths.patch

```diff
@@ -0,0 +1,12 @@
+diff --exclude '*~' -ruN a/gmsh.py b/gmsh.py                                                         
+--- a/gmsh.py
++++ b/gmsh.py
+@@ -25,7 +25,7 @@
+ __version__ = GMSH_API_VERSION
+ 
+ oldsig = signal.signal(signal.SIGINT, signal.SIG_DFL)
+-libdir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Please use ``substitute --replace 'in' 'out'.

Reply:
I don't follow, can you elaborate?